packages/clerk-js/src/ui/customizables/index.ts

```diff
@@ -32,6 +32,12 @@ export const Image = makeCustomizable(sanitizeDomProps(makeResponsive(Primitives
 export const Alert = makeCustomizable(sanitizeDomProps(Primitives.Alert));
 export const AlertIcon = makeCustomizable(sanitizeDomProps(Primitives.AlertIcon));
 export const Input = makeCustomizable(sanitizeDomProps(Primitives.Input), { defaultDescriptor: descriptors.input });
+export const CheckboxInput = makeCustomizable(sanitizeDomProps(Primitives.Input), {
+  defaultDescriptor: descriptors.checkboxInput,
+});
```

Comment:
I'd expect that this component includes properties that come with a checkbox, e.g: `type='checkbox'`

Reply:
We can't pass props like `type="checkbox"` here, the `makeCustomizable` accepts only some generic options `defaultDescriptor` and `defaultStyles`.